js/formValidation.js

```diff
@@ -0,0 +1,43 @@
+Fliplet.Env.set('appId', 1);
+
+var android;
+var requiredList;
+
+function validateForm(platform, submissionType){
+	return new Promise(function (resolve, reject){
```

Comment:
I mean, I'm fine if you want to return a promise and there's a reason for it :) but it does seem to be all sync, plus you're never resolving it.